BallBounceGame/Assets/SettingsGUI/SettingsScript.cs

```diff
@@ -35,8 +45,54 @@ void Start()
         if (PlayerPrefs.GetInt("Sound", 1) == 0)
             SFXButtonPressed();
 
+        if (PlayerPrefs.GetInt("DebugDraw", 0) == 1)
+            DebugDrawButtonPressed();
+
+        fps_lock = PlayerPrefs.GetInt("FPSLock",60);
 
+        if (fps_lock == 1000)
+            fps_lock_button.GetComponentInChildren<Text>().text = "UNLOCKED";
+        else
+            fps_lock_button.GetComponentInChildren<Text>().text = fps_lock.ToString();
+
+    }
+    public void FPSButtonPressed()
+    {
+  
+        // Handling cases of custom fps locks manually set in the config file
+        bool found = false;
+        for(int i=0; i<fps_list.GetLength(0); i++)
+        {
+            
+            if (fps_lock == fps_list[fps_list.GetLength(0)-1])
```

Comment:
Interesting special case, would probably be ok to have this as a separate boolean (<int>fps [if locked], <boolean>fpsLock)

Reply:
Since we do `Application.targetFrameRate = fps_lock`, Unity's actual frame rate is set to 1000, which is essentially unlocking it. We don't want to display 1000 in the menu so it checks for this special case. But using 1000 is functional, not just a placeholder.